lib/util/git/private-key.spec.ts

```diff
@@ -10,39 +13,121 @@ jest.mock('fs-extra', () =>
     >('../../../test/fixtures')
     .fsExtra(),
 );
-jest.mock('../exec');
+jest.mock('../exec', () =>
+  jest
+    .requireActual<typeof import('jest-mock-extended')>('jest-mock-extended')
+    .mockDeep(),
+);
```

Comment:
```suggestion
jest.mock('../exec', () => mockDeep());
```

This works on other tests

Reply:
Done: adb96b8a563553ae2e404e71a45781e933cce1f5